tasks/edge.rake

```diff
@@ -109,7 +109,6 @@ namespace :edge do
         Bundler.with_unbundled_env do
           puts "======== Updating #{integration} in #{gemfile} ========\n"
           output, = Open3.capture2e({ 'BUNDLE_GEMFILE' => gemfile.to_s }, "bundle lock --update=#{gem}")
-
```

Comment:
super nitpicky, but can probably undo this just so it removes a file from being modified here.